content/guides/advanced/verified-builds.md

```diff
@@ -232,7 +276,14 @@ solana-verify get-program-hash -u $NETWORK_URL $PROGRAM_ID
 > You may have different versions deployed on different
 > [Solana clusters](/docs/core//clusters.md) (i.e. devnet, testnet, mainnet).
 > Ensure you use the correct network URL for the desired Solana cluster you want
-> to verify a program against.
+> to verify a program against. Remote verification will only work on mainnet.
+
+Now you can already get the hash of your program and compare it to your binary
+hash from earlier if you want:
+
+```bash
+solana-verify get-program-hash yourProgramId
```

Comment:
since the above command uses `$PROGRAM_ID` as the filler value, lets be consistent and use the same one:
```suggestion
solana-verify get-program-hash $PROGRAM_ID
```